modules/series-service-impl/src/main/java/org/opencastproject/series/endpoint/SeriesRestService.java

```diff
@@ -1737,6 +1737,55 @@ public Response getSeriesElement(
     }
   }
 
+  @PUT
+  @Path("{seriesId}/extended-metadata/{type}")
+  @RestQuery(
+          name = "updateExtendedMetadata",
+          description = "Updates extended metadata of a series",
+          returnDescription = "An empty response",
+          pathParameters = {
+                  @RestParameter(name = "seriesId", description = "The series identifier", type = STRING,
+                          isRequired = true),
+                  @RestParameter(name = "type", description = "The catalog type", type = STRING,
```

Comment:
Might be good to give an example for `type`. Also, is type a unique identifier?

Reply:
Thanks for that. Endpoints with type fields that don't specify what `type` refers to is a little pet peeve of mine (there's a couple more of those in Opencast).